macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -172,11 +172,13 @@
         'owners': elementary.insensitive_get_dict_value(test_node, 'model_owners'),
         'tags': elementary.insensitive_get_dict_value(test_node, 'model_tags'),
         'test_results_query': test_results_query,
-        'test_name': elementary.insensitive_get_dict_value(test_node, 'short_name'),
+        'test_name': elementary.insensitive_get_dict_value(test_node, 'name'),
```

Comment:
Is there a compatibility issue here? 
E.g. if people update the DBT package but not the CLI this can cause a behavior change, no?
Though in any case you mentioned we're increasing the minor version.

Reply:
There is no problem with it because `name` is a field that is already on the `test_node` object, and we just replaced how we create the `test_name` key (without creating any new thing).

And yes, the version bumping will make us safe anyway!